src/main/java/de/qabel/desktop/inject/RuntimeDesktopServiceFactory.java

```diff
@@ -219,13 +219,15 @@ public synchronized SyncDaemon getSyncDaemon() {
     @Override
     public synchronized DropDaemon getDropDaemon() {
         if (dropDaemon == null) {
+            /*
```

Comment:
It executes like `new Thread(null).start();` so the global thread-exceptionhandler silently catches it.
Looks like we need a super-integrationtest that actually starts everything together with the real DI container ... just for you XD